zgw_consumers/settings.py

```diff
@@ -4,9 +4,9 @@
 from .constants import NLXDirectories
 
 NLX_DIRECTORY_URLS = {
-    NLXDirectories.demo: "https://directory.demo.nlx.io/",
+    NLXDirectories.demo: "https://demo-directory-api.commonground.acc.utrecht.nl/",
     NLXDirectories.preprod: "https://directory.preprod.nlx.io/",
```

Comment:
shouldn't we just remove this one then instead? it's not available either way

Reply:
removed it